rosidl_generator_cpp/resource/srv.hpp.template

```diff
@@ -12,6 +12,7 @@
 #ifndef __@(spec.pkg_name)__srv__@(get_header_filename_from_msg_name(spec.srv_name))__hpp__
 #define __@(spec.pkg_name)__srv__@(get_header_filename_from_msg_name(spec.srv_name))__hpp__
 
+#include "@(spec.pkg_name)/srv/@(get_header_filename_from_msg_name(spec.srv_name))__traits.hpp"
```

Comment:
The service struct only references the request and reponse message types. Why do they need a trait?